packages/strapi-design-system/src/v2/SimpleMenu/SimpleMenu.js

```diff
@@ -6,11 +6,12 @@ import { Typography } from '../../Typography';
 import { Box } from '../../Box';
 import { Flex } from '../../Flex';
 import { Button } from '../../Button';
-import { BaseLink } from '../../BaseLink';
 import { Popover, POPOVER_PLACEMENTS } from '../../Popover';
 import { getOptionStyle } from './utils';
 import { useId } from '../../helpers/useId';
 import { KeyboardKeys } from '../../helpers/keyboardKeys';
+import { Link } from '../../Link';
+import { NavLink } from 'react-router-dom';
```

Comment:
`NavLink` shouldn't be included in the `v2` components - the idea of v2 components was removing the dependency to `react-router-dom` as it is not compatible with other routing systems (e.g. Next)

Reply:
Noted.